### DISCUSSION TITLE
Chis .temp and .plot speed transfer

### DISCUSSION THREAD

**Discussion by grga1962** [Chia-Network/chia-blockchain#5489] [General]
Can the transfer of a temporary file to a permanent one be accelerated. For me, it takes 2-5 hours for 6 plots?

**Comment by SackofFluor** [comment]
Well it only depends on the transferspeed between your devices....

What you could do, for example, use a "semi"-permanent storage as a buffer in between. eg. 1x2TB HDD in Raid0. Then the SSD from the plot will be "removed" much faster into the semi-permanent storage.

Also, copying multiple plots at the same time in parallel can (depending on HOW you do it) be extremely inefficient.

Most people (like me) use a NAS for the permanent storage.... in this case my transfer speed is limited by my network speed. Any upgrade on this side becomes expensive very fast.

One way (which would involve a lot of manual labor though) would be:
SSD(temp) ---> Raid0 HDD(semi-perm) -->USB-C to a JBOD -->occasionally (once a day or so) physically connect the JBOD to a USB-C on your NAS...
That could maybe increase copying speeds...

Only other alternative:
Directly attach a metric-sh***ton of HDDs directly via SATA to your Motherboard.

**Comment by grga1962** [comment]
Temporary files are located on the NVMe disk. I save permanent files to an SSD. Later I move them from the SSD to the HDD. This transfer from NVMe to SSD takes a long time. How to speed it up. The transfer process from NVm to SSD takes 2-6 hours. Until this process is complete I cannot start a new plotting session

**Reply by SackofFluor** [reply]
then the actual transfer speed within your system is just slow...
maybe an overcrowded PCIE-Lane.
Maybe a filled buffer, maybe just a bad SSD?
